docs/5g-on-helium/mining-5g.mdx

```diff
@@ -31,11 +31,17 @@ Note that connecting more than one Small Cell Radio will require an additional n
 Refer to the 5G Hotspot Maker installation guides for specific instructions.
 
 Instructions on installing a CBRS Small Cell Radio and registration with the SAS are provided by the
-individual 5G Hotspot Makers. As an example, see the FreedomFi installation guides here:
+individual 5G Hotspot Makers.
+
+For FreefomFi Gateways refer to the installation guides here:
 
 - [FreedomFi Indoor Install Guide](https://freedomfi.com/freedomfi-indoor-cbrs-small-cell-setup-guide/)
 - [FreedomFi Outdoor Install Guide](https://freedomfi.com/freedomfi-outdoor-cbrs-small-cell-setup-guide/)
 
+For Bobcat500 Gateways refer to the Bobcat Knowledge Center:
+
+- [Bobcat Knowledge Center CBRS Radio Troubleshooting Guide](https://bobcatminer.zendesk.com/hc/en-us/articles/7282743642395-Troubleshooting-Issues-with-the-Indoor-CBRS-Small-Cell-Radio)
```

Comment:
Maybe a more generic link will be better? https://bobcatminer.zendesk.com/hc/en-us/articles/4588536001435-Overview